lib/utils/hasPrefix.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { prefixes } = require('../reference/prefixes');
+
+const HAS_PREFIX_REGEX = new RegExp(`(?:${[...prefixes.values()].join('|')})`, 'i');
```

Comment:
[ask] Is omitting `^` (starts with) intentional, right? Acceptable?

Reply:
We may test broader strings, e.g. a selector list where the first selector isn't prefixed.

Reply:
I see. I'm a bit concerned that `value-no-vendor-prefix` is using `startsWith`. 🤔

Reply:
Ah, I misunderstood! `value-no-vendor-prefix` checks a value by `isAutoprefixable.propertyValue()`. Okay, no problem. 👍🏼